### ISSUE TITLE
Make sstableloader incremental

### ISSUE THREAD

**Issue by haaawk** [scylladb/scylla-tools-java#64]
Currently when the process of loading sstables is interrupted, we have to start over. It would be great if the process could pick up where it finished and continue without doing the same work again.

**Comment by haaawk** [comment]
@slivne @elcallio @avikivity

**Comment by elcallio** [comment]
Note that this is not so easy: 
We both have to assume that the source file tree is readonly (and should be always treated as such), and we also have to take into account the various options that control whether a file can really be considered "processed", such as initial and subsequent connected nodes and other parameters. 

Perhaps adding a "progress" output file, and the option to restart from it would be better.

**Comment by haaawk** [comment]
Yes. I didn't mean that it has to always be incremental. I should have stated it better. Progress file and option to use it is exactly what we should do IMO.
